services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:

```

Comment:
The function `get_cinema_halls` should have a return type of `QuerySet` instead of `None`. Update the function signature to `def get_cinema_halls() -> QuerySet:`.